pkg/cmd/helmfile/resolve/resolve.go

```diff
@@ -327,12 +330,12 @@ func (o *Options) saveNamespaceJXValuesFile(helmfileDir string, ns string) error
 
 func (o *Options) upgradeHelmfileStructure(dir string) (int, error) {
 	count := 0
-	increment, err := o.processHelmfile(o.Helmfiles[0])
+	err := o.processHelmfile(o.Helmfiles[0])
 	if err != nil {
 		return 0, errors.Wrapf(err, "error processing parent helmfile before restructure")
 	}
 
-	count += increment
+	count += 0
```

Comment:
This is always 0, will open another PR to refactor, need to take a deeper :eyes: